packages/scheduler/src/Scheduler.js

```diff
@@ -422,7 +407,7 @@ export {
   unstable_cancelCallback,
   unstable_wrapCallback,
   unstable_getCurrentPriorityLevel,
-  unstable_shouldYield,
+  shouldYieldToHost as unstable_shouldYield,
```

Comment:
This'll be a breaking for the next sync.

Reply:
How so? The export is named the same

Reply:
nvm I read as being removed 💀